sorted_words.rb

```diff
@@ -0,0 +1,15 @@
+words = []
+
+while true do 
```

Comment:
I try to avoid using `while true` loops, because then if any of the branches within that loop do not `break` correctly you end up with an infinite loop, which is no good